### DISCUSSION TITLE
how to apply the handycontrol theme to a custom datagrid?

### DISCUSSION THREAD

**Discussion by FardinFarnezhad** [HandyOrg/HandyControl#732] [Q&A] [answered]
hey i have to use a custom data grid for multi selection and i has the ugly white theme while my other normal datagrids are using handycontrol beutifull theme? how can i apply the theme to my custom datagrid too?

**Comment by ghost1372** [comment, marked as answer]
Hi @FardinFarnezhad  
Use **BasedOn** Property to define styles

```
<Style x:Key="myCellStyle" BasedOn={StaticResource DataGridCellStyle} TargetType="DataGridCell">

</Style>
```

[Here ](https://github.com/HandyOrg/HandyControl/blob/master/src/Shared/HandyControl_Shared/Themes/Styles/DataGrid.xaml) you can find the style name of each part
